model_compiler/README.md

```diff
@@ -247,50 +271,52 @@ Usage: anntest <weights.bin> [<input-data-file(s)> [<output-data-file(s)>]]]
 ...
 % ./anntest ../weights.bin input-%04d.png output-%04d.png,reference.rgb,0.01
 ...
-````
+```
 
 Test code with preprocessing add / multiply values to normalize the input tensor. Some models(e.g. Inception v4) require input tensor to be normalized. You can pass the preprocessing values using --add & --multiply option.
 
-````
+``` 
 % ./anntest ../weights.bin input.f32 output.f32 --add -2.1,-2.07,-1.8 --multiply 0.017,0.017,0.017
 ...
-````
+```
 
 ## Models & Operators currently supported
+
 ###  Models
 
 <p align="center"><img width="60%" src="../docs/images/modelTrainedFrameWorks.png" /></p>
 
 |Networks|Caffe|ONNX|NNEF|
 |--------|-----|----|----|
-|AlexNet||&#9745;|&#9745;|
-|Caffenet||&#9745;||
-|DenseNet||&#9745;||						
-|Googlenet|&#9745;|&#9745;|&#9745;|		
-|Inception-V1||&#9745;|&#9745;|			
-|Inception-V2||&#9745;|&#9745;|			
+|AlexNet||&#9745; |&#9745; |
```

Comment:
Codacy found an issue: [[table-cell-padding] Cell should be compact](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6056700)